vendor/modules.txt

```diff
@@ -168,10 +168,6 @@ k8s.io/api/apps/v1
 k8s.io/api/apps/v1beta1
```

Comment:
is updating this file necessary? it looks like just some lines got moved around?

Reply:
Yes content is the same, just some order changes, but I'm getting error from CI like below when I don't change this file with the below command;

`ERROR: vendor directory *not* up-to-date, it did get modified by 'GO111MODULE=on go mod vendor':`

https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-csi_external-provisioner/327/pull-kubernetes-csi-external-provisioner-unit/1187927085243437058

https://travis-ci.org/kubernetes-csi/external-provisioner/builds/603113049?utm_source=github_status&utm_medium=notification